fonts/other/rubik-one/scss/mixins.scss

```diff
@@ -0,0 +1,184 @@
+@use 'sass:list';
+@use 'sass:map';
+@use 'sass:math';
+@use 'sass:meta';
+@use 'sass:string';
+
+@use 'metadata';
+
+$metadata: meta.module-variables(metadata) !default;
+$directory: null !default;
+
+$family: null !default;
+$display: null !default;
+$formats: null !default;
+$subsets: null !default;
+$weights: null !default;
+$styles: null !default;
+$axes: null !default;
+
+// Deprecated
+$displayVar: null !default;
+
+@mixin generator(
+  $metadata: $metadata,
+  $directory: $directory,
+  $family: $family,
+  $display: $display,
+  $formats: $formats,
+  $subsets: $subsets,
+  $weights: $weights,
+  $styles: $styles,
+  $axes: $axes,
+
+  // Deprecated
+  $displayVar: $displayVar
+) {
+  @if $displayVar != null {
+    @warn "$displayVar is deprecated due to the limitation of using css variables in @font-face (https://github.com/fontsource/fontsource/issues/726).";
+  }
+
+  $isVariable: map.get($metadata, axes) != null;
+
+  $directory: if(
+    $directory,
+    $directory,
+    '~@fontsource#{if($isVariable, '-variable', '')}/#{map.get($metadata, id)}/files'
+  );
+
+  $family: if($family, $family, map.get($metadata, family) + if($isVariable, ' Variable', ''));
+  $display: if($display, $display, swap);
+  $formats: if(not $formats or $formats == all, if($isVariable, woff2, (woff2, woff)), $formats);
+  $subsets: if(
+    $subsets,
+    if($subsets == all, map.get($metadata, subsets), $subsets),
+    map.get($metadata, defaults, subset)
+  );
+  $weights: if(
+    $weights,
+    if($weights == all, map.get($metadata, weights), $weights),
+    map.get($metadata, defaults, weight)
+  );
+  $styles: if(
+    $styles,
+    if($styles == all, map.get($metadata, styles), $styles),
+    map.get($metadata, defaults, style)
+  );
+  $axes: if(
+    $axes,
+    if($axes == all, full, $axes),
+    if($isVariable, if(map.has-key($metadata, axes, wght), wght, full), null)
+  );
+
+  @each $subset in $subsets {
+    @each $unicodeSubset, $unicodeRange in map.get($metadata, unicode) {
+      @if (
+        ($subset == $unicodeSubset) or
+          (
```

Comment:
Can you regenerate this package with the latest commit in this repo or the latest version of `@fontsource-utils/cli`. We pushed a bugfix for this file.

Reply:
Updated :)